pages/contact.tsx

```diff
@@ -8,23 +8,29 @@ import Link from "next/link";
 import {useState} from "react";
 
 const mailTo: string = "robert@odellinsuranceagency.com";
-const mailSubject = "Sent From odellinsuranceagency.com";
-const crlf = "%0d%0a";
+const subject = "Sent From odellinsuranceagency.com";
+const crlf = "\r\n";
+
+function getMailLink(name: string, email: string, phone: string,): string {
```

Comment:
Not sure why the linting isn't catching this, but there is a trailing comma after the last parameter that shouldn't be there.

Reply:
removed extra comma